test/spec/asyncapi.yml

```diff
@@ -173,6 +173,9 @@ components:
     lightMeasuredPayload:
       type: object
       properties:
+        testProp: 
```

Comment:
can we use dummy file in test, like with HTML template https://github.com/asyncapi/html-template/blob/master/.github/workflows/pr-testing-with-generator.yml#L25 so we basically make sure first that dummy file has this new attribute there?

Reply:
We could, is it better it is an entirely new workflow as well or is it fine it reside as an npm script 🤔 ?

Reply:
I'm definitely with npm script, the way I did with like this in HTML was because there was this bug with installing templates that I solved after creating the workflow

Reply:
Added a PR to the generator since none of the objects used additionalProperties - https://github.com/asyncapi/generator/pull/578